infrastructure/lambda/template.yaml

```diff
@@ -386,7 +390,11 @@ Resources:
       Environment:
         Variables:
           POWERTOOLS_SERVICE_NAME: !Sub "${CriIdentifier}-postcode-lookup"
-          SQS_AUDIT_EVENT_QUEUE_URL: !ImportValue AuditEventQueueUrl
+          SQS_AUDIT_EVENT_QUEUE_URL:
+            Fn::ImportValue:
+              !Sub
+                - ${stackName}-AuditEventQueueUrl
+                - stackName: !If [IsDevEnvironment, !Ref CommonStackName, !Ref TxmaStackName]
```

Comment:
Do when @CharlesIC has moved the queues into the txma-infra repo we should be able to remove this, right? We will always be using the `TxmaStackName`?

Reply:
Yes, that's correct